neo/Cryptography/ECC/ECDsa.cs

```diff
@@ -7,18 +7,25 @@ namespace Neo.Cryptography.ECC
 {
     public class ECDsa
     {
-        private readonly byte[] privateKey;
         private readonly ECPoint publicKey;
         private readonly ECCurve curve;
+        private readonly BigInteger d;
 
         public ECDsa(byte[] privateKey, ECCurve curve)
```

Comment:
This constructor of ecdsa may cause private key leak because it depends on the ECC mulitple operation.

Actually there are some functions in neo-cli use ECPoint.Multiply,too. They are also risky.

Reply:
?